tests/components/mqtt/test_cover.py

```diff
@@ -587,7 +606,8 @@
         },
     )
 
-    assert hass.states.get("cover.test").attributes["supported_features"] == 251
+    assert hass.states.get(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)